### DISCUSSION TITLE
Border less Design ?

### DISCUSSION THREAD

**Discussion by adityatelange** [adityatelange/hugo-PaperMod#183] [Ideas]
I was thinking of removing the **borders** from **Table of Contents** as well as from **share buttons** and **Tags** under single page layout.

What do you guys think ? Let me know... 
I'll be putting some screenshots for side by side comparison 

This is added to my blog website, [here](https://adityatelange.me/blog/hugo-external-link-new-tab/)

---

This concept is being thought of to improve #180 's integration with the theme (currently it looks misplaced imo) and overall lookalike-ness (also referring #101)

![image](https://user-images.githubusercontent.com/21258296/104930982-d57e7600-59cb-11eb-8919-7ff11b7fd4c3.png)

**Comment by adityatelange** [comment]
# Table of Contents

## Before

<kbd>![image](https://user-images.githubusercontent.com/21258296/104930188-d236ba80-59ca-11eb-9d94-c5cab3873079.png)
</kbd>

## After

<kbd>![image](https://user-images.githubusercontent.com/21258296/104930031-a1ef1c00-59ca-11eb-9e70-f538d15e39d6.png)
</kbd>

**Comment by adityatelange** [comment]
# Tags and Share Buttons

## Before

<kbd>![image](https://user-images.githubusercontent.com/21258296/104930564-4ffac600-59cb-11eb-971c-da5c6906a1dd.png)
</kbd>


## After
<kbd>![image](https://user-images.githubusercontent.com/21258296/104930603-6012a580-59cb-11eb-9a8f-a3f8f17c4ad2.png)
</kbd>

**Comment by kdkasad** [comment]
I actually prefer the borders. Maybe this should be turned into a config toggle?

**Reply by pixelfabs** [reply]
> I actually prefer the borders. Maybe this should be turned into a config toggle?

Me too. It did help me a lot when starting to customize stuff. Normally you have to add border: 1px solid black to everything, just to see what was happening. In this case, I could just start and remove it when done.

**Comment by adityatelange** [comment]
> I actually prefer the borders. Maybe this should be turned into a config toggle?

Ya me too, but the borders does not match the theme you know.
The more optional stuff we add, more there will be unused css affecting performance directly.

**Reply by danielfdickinson** [reply]
In my sites (I can include them here if you want to look), my colour scheme (I override the defaults) ends up looking borderless anyway and I prefer the borderless look.
